library/common/extensions/filters/http/platform_bridge/filter.h

```diff
@@ -114,9 +119,13 @@ class PlatformBridgeFilter final : public Http::PassThroughFilter,
     IterationState iteration_state_;
     PlatformBridgeFilter& parent_;
     envoy_filter_on_headers_f on_headers_;
+    bool on_headers_called_{};
```

Comment:
I'd actually be more curious if these have been _forwarded_ than called.

In other words, you'd have to have continued/resumed iteration and have them not be pending.

Reply:
I think having iteration_state + on_resume_called + the state of buffer/pending_<headers/trailer> does the trick. wdyt?